maintenance_scripts/Dockerfile

```diff
@@ -5,9 +5,11 @@ RUN useradd -m user && \
     echo "root ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
     echo "user ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
 
-RUN pacman -Syu --noconfirm --needed sudo binutils python-pylint
+RUN pacman -Syu --noconfirm --needed sudo binutils make python-pylint
```

Comment:
i think it should go to `makedepends`

Reply:
I checked on the Arch docs/forums. make is part of base-devel which is assumed to be installed to use AUR package. So I guess this would go against the guidelines. Can add it anyway though, costs nothing.

Reply:
for sure it's better to adhere the guidelines

Reply:
mb then install here `base-devel` instead of make and binutils?

Reply:
Yeah I'll do that. For reference by the way:
> Note: The group [base-devel](https://www.archlinux.org/groups/x86_64/base-devel/) is assumed to be already installed when building with makepkg. Members of this group should not be included in makedepends array.
> — https://wiki.archlinux.org/index.php/PKGBUILD#makedepends